### PR TITLE
finished menu page

### PR DESCRIPTION
**Pull request by fattyduck**

### CODE REVIEW

File: MemeifyMe/app/src/main/java/accesscode/c4q/nyc/memeifyme/MainActivity.java [hunk 1]
```diff
@@ -20,16 +25,33 @@ protected void onCreate(Bundle savedInstanceState) {
         setContentView(R.layout.activity_main);
 
         camera_button = (Button) findViewById(R.id.camera_button);
```
Comment (jaellys) [inline MemeifyMe/app/src/main/java/accesscode/c4q/nyc/memeifyme/MainActivity.java:27]: I think we should stick to a naming convention of general -> specific. So these buttons, for example, would be button_camera, button_album, and so on. I've also seen btn used, particularly in XML.

File: MemeifyMe/app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -9,9 +9,52 @@
...
     tools:context=".MainActivity">
 
+
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:textSize="40dp"
+        android:layout_marginTop="20dp"
+        android:textAppearance="?android:attr/textAppearanceLarge"
+        android:text="Memifier"
```
Comment (jaellys) [inline MemeifyMe/app/src/main/res/layout/activity_main.xml:19]: Let's minimize hard-coded strings by keeping these resources in our strings.xml, yeah?

File: MemeifyMe/app/src/main/res/layout/activity_main.xml [hunk 3]
```diff
@@ -9,9 +9,52 @@
     android:paddingTop="@dimen/activity_vertical_margin"
     tools:context=".MainActivity">
 
+
```
Comment (jaellys) [inline MemeifyMe/app/src/main/res/layout/activity_main.xml:12]: As a general rule, please remember to run Code > Re-arrange code in order to keep everything organized and consistent. Thanks!

File: MemeifyMe/app/src/main/res/layout/activity_main.xml [hunk 4]
```diff
@@ -9,9 +9,52 @@
...
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:textSize="40dp"
+        android:layout_marginTop="20dp"
+        android:textAppearance="?android:attr/textAppearanceLarge"
+        android:text="Memifier"
+        android:id="@+id/titleTextView"
```
Comment (jaellys) [inline MemeifyMe/app/src/main/res/layout/activity_main.xml:20]: I definitely prefer the snake_case used elsewhere in this XML -- should we stay in line with that? Also, not to broken record it up, but let's name general -> specific (ex: tv_title).

File: MemeifyMe/app/src/main/java/accesscode/c4q/nyc/memeifyme/MainActivity.java [hunk 5]
```diff
@@ -20,16 +25,33 @@ protected void onCreate(Bundle savedInstanceState) {
...
+        template_button.setOnClickListener(this);
+        exit_button.setOnClickListener(this);
 
-            }
-        });
 
+    }
 
+    @Override
+    public void onClick(View view) {
```
Comment (jaellys) [inline MemeifyMe/app/src/main/java/accesscode/c4q/nyc/memeifyme/MainActivity.java:41]: Good! Thanks for taking care of this. :]

**Comment by hyunj0** [comment]
Saw a live demo of the menu and it looks great minus the exit so let's change that!

**Pull request merged**